idd/Energy+.idd.in

```diff
@@ -33653,13 +33653,6 @@ ZoneHVAC:PackagedTerminalHeatPump,
         \default 0.001
         \units dimensionless
         \note Defines Heating convergence tolerance as a fraction of Heating load to be met.
-   N8 , \field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation
-        \type real
-        \minimum -20.0
-        \default -8.0
-        \units C
-        \note Needs to match the corresponding minimum outdoor temperature defined
-        \note in the DX Heating Coil object.
```

Comment:
It means more example file changes, more unit test changes and changing the code. Can I pick this one up with the next issue of the parent using the same OA condition as the child? Or do it now?